redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -211,6 +211,99 @@ object Input {
       )
   }
 
+  case object IdleInput extends Input[Idle] {
+    def encode(data: Idle): Chunk[RespValue.BulkString] =
+      Chunk(stringEncode("IDLE"), stringEncode(data.millis.toString))
+  }
+
+  case object TimeInput extends Input[Time] {
```

Comment:
Can this be done as an `Input[Duration]`?

Reply:
You're right. `IdleInput` too. There is a lot of stuff like this all over the project left from before usability PR was merged, so it will probably require separate PR to resolve it. I will check other input types related to streams too and remove what's unnecessary here.